app/views/users/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h3> Your Events </h3>
+<p>
+   <ul id = "users">
+      <% @user.created_events.each do |event|%>
+      <li>
+          Event: <%= event.name %> <br/> 
+      </li>
+      <li>
+        Location: <%= event.location %><br/>
+      </li>
+      <li>     
+         Event_Date: <%= event.start_date %>
+      </li>
+      <%end%>    
+   </ul> 
+</p>
+<h3>Future Events</h3>
+<ul>
+   <%@user.attended_events.each do |event| %>
```

Comment:
This logic should be in the controller, e.g. `@past_events = current_user.attended_events.previous` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations)). Then you can use `@past_events.each` in your view.